src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexBoyerMoore.cs

```diff
@@ -18,29 +17,25 @@ namespace System.Text.RegularExpressions
 {
     internal sealed class RegexBoyerMoore
     {
-        internal int[] _positive;
-        internal int[] _negativeASCII;
-        internal int[][] _negativeUnicode;
-        internal String _pattern;
-        internal int _lowASCII;
-        internal int _highASCII;
-        internal bool _rightToLeft;
-        internal bool _caseInsensitive;
-        internal CultureInfo _culture;
```

Comment:
The following fields aren't used anywhere else and can be `private`
 - `_rightToLeft`
- `_culture`